tests/test_completion.py

```diff
@@ -8,6 +8,7 @@
 from typing import List
 
 import pytest
+from packaging import version
```

Comment:
that's a tricky one. Packaging is available because of the pytest dependency.
```
$ pipdeptree  -r -p packaging
packaging==20.3
  - pytest==5.4.1 [requires: packaging]
```

The pytest dependency is a test only one, so this will not work at runtime.
It's fine, but I was surprised to see you use it here without having to add the dependency.

Reply:
Hmm, I thought it was a built-in module. I'll add that dependency in dev requirements.

Reply:
sounds good.